modules/asg_node_group/main.tf

```diff
@@ -120,6 +120,13 @@ resource "aws_launch_template" "config" {
     }
   }
 
+  metadata_options {
+    http_endpoint               = "enabled"
+    http_tokens                 = var.nodes_metadata_http_tokens
+    http_put_response_hop_limit = 1
+    instance_metadata_tags      = "disabled"
```

Comment:
are we sure nothing depends on this?

Reply:
or does it default to "disabled" anyway?

Reply:
instance_metadata_tags      = "disabled".  Is the default option
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_template
is a new feature of  Jan 6, 2022 so very likely no cluster uses it, because in the EKS module there is no refference to this variable.
https://aws.amazon.com/about-aws/whats-new/2022/01/instance-tags-amazon-ec2-instance-metadata-service/

WDYT @errm ?

Reply:
OK makes sense... I didn't know it was a very new feature ...